valkey.conf

```diff
@@ -329,7 +329,7 @@ daemonize no
 #
 # When the server runs non daemonized, no pid file is created if none is
 # specified in the configuration. When the server is daemonized, the pid file
-# is used even if not specified, defaulting to "/var/run/redis.pid".
+# is used even if not specified, defaulting to "/var/run/valkey.pid".
```

Comment:
This is not an URL.

The default is controlled by a macro in server.h, which is still defined like this:

    #define CONFIG_DEFAULT_PID_FILE "/var/run/redis.pid"

The comments here better match the actual default.

Reply:
Yes, this is the only remaining reference to redis that should exist in the valkey.conf file.

Reply:
We mentioned that changing this default might be considered a breaking change, so we'll do it in the next major version. I created this now: #321.

Reply:
Thanks for the reminder, @zuiderkwast